src/core/include/openvino/opsets/opset.hpp

```diff
@@ -136,4 +136,5 @@ const OPENVINO_API OpSet& get_opset5();
 const OPENVINO_API OpSet& get_opset6();
 const OPENVINO_API OpSet& get_opset7();
 const OPENVINO_API OpSet& get_opset8();
+const OPENVINO_API OpSet& get_opset_by_name(const std::string& opset_name);
```

Comment:
Do we need this helper in public API?

Reply:
discussed offline
this helper will be moved for python bindings only

Reply:
the helper has been moved to frontend/extensions/op.hpp